geemusic/utils/music_queue.py

```diff
@@ -69,7 +69,7 @@ def reset(self, tracks=[]):
         else:
             return self.song_ids[self.current_index]
 
-    def enqueue_track(self, song):
+    def enqueue_track(self, song, song_id):
```

Comment:
why song_id?

Reply:
Accidentally put that there... It has been removed.